### PR TITLE
Swift 4.2

### PR DESCRIPTION
**Pull request by volatilegg**
Breaking changes:
- Remove redundant property `String.length`, using `String.count` instead 

All the tests have passed running on XCode 10

### CODE REVIEW

File: QvikSwift/Keychain.swift [hunk 1]
```diff
@@ -26,6 +26,8 @@ import Foundation
  A generic keychain wrapper utility. For information on the keychain concept, see:
  `https://developer.apple.com/library/content/documentation/Security/Conceptual/keychainServConcepts/02concepts/concepts.html`
  */
+
+// swiftlint:disable identifier_name
```
Comment (matti777) [inline QvikSwift/Keychain.swift:30]: Why is this needed?

Comment (volatilegg) [reply QvikSwift/Keychain.swift:30]: Good question, I got an error from Swiftlint out of no where `Enum element name should only contain alphanumeric characters: 'apiCallFailed(statusCode:)` 

screen shot 2018-07-04 at 10 43 55

The enum looks fine for me and swiftlint is updated to latest version `0.26.0`, hence I chose to ignore the rule.

Here's the lint rule
https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/IdentifierNameRule.swift
